lib/teleterm/api/uri/uri_test.go

```diff
@@ -81,13 +76,49 @@ func TestParseClusterURI(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		tt := tt
 		t.Run(tt.in, func(t *testing.T) {
-			t.Parallel()
-
 			out, err := uri.ParseClusterURI(tt.in)
 			require.NoError(t, err)
 			require.Equal(t, tt.out, out)
 		})
 	}
 }
+
+func TestGetDbName(t *testing.T) {
+	tests := []struct {
+		in  uri.ResourceURI
+		out string
+	}{
+		{
+			uri.NewClusterURI("foo").AppendDB("postgres"),
+			"postgres",
+		},
+		{
+			uri.NewClusterURI("foo").AppendLeafCluster("bar").AppendDB("postgres"),
+			"postgres",
+		},
+		{
+			uri.NewClusterURI("foo"),
+			"",
+		},
+		{
+			uri.NewClusterURI("foo").AppendLeafCluster("bar"),
+			"",
+		},
+		{
+			uri.NewClusterURI("foo").AppendKube("k8s"),
+			"",
+		},
+		{
+			uri.NewClusterURI("foo").AppendLeafCluster("bar").AppendKube("k8s"),
+			"",
+		},
```

Comment:
Can we add a test name description ? 
```
	tests := []struct {
               name string
		in  uri.ResourceURI
		out string
	}{
            name: "should return postgres name" 
```

This will allow the reader to better understand what and why to except.